src/OpenSearch.OpenSearch.Managed/ConsoleWriters/LineOutParser.cs

```diff
@@ -117,7 +117,6 @@ private LineOutParser(string shortNamePrefix, string fullNamePrefix, string secu
 			return true;
 		}
 
-
```

Comment:
For future reference, generally we should keep whitespace reformatting to be in a separate PR from actual code changes so that the review doesn't get a mix between whitespace changes and actual code changes (for readability).

Reply:
For sure. This caused by manual merge conflict resolution, I removed empty line considered that it was added by git.